source/tutorials/multi-boot/dual-boot-win.rst

```diff
@@ -0,0 +1,235 @@
+
+Method 1: Shrink Windows partition and install |CL|
+***************************************************
+
+#. Boot up the Windows 10 OS.
+
+#. Launch the :file:`Disk Management` utility found under
+   :guilabel:`Create and format hard disk partitions`.
+
+#. Right-click the primary Windows partition and select :guilabel:`Shrink Volume...`.
+
+   .. figure:: ../../_figures/multi-boot/multi-boot-win-01.png
+      :width: 45%
+      :alt: Disk Management > Shrink Volume
+
+      Figure 1: Disk Management > Shrink Volume.
+
+#. Shrink the size of the partition by at least the following amount:
+
+   * For :ref:`installing the live desktop <bare-metal-install-desktop>`,
+     allow at least 21GB.
+
+   * For :ref:`installing the live server <bare-metal-install-server>`,
+     allow at least 4GB.
+
+
+#. We shrink C by about 21 GB, as shown in Figure 2.
+
+   .. figure:: ../../_figures/multi-boot/multi-boot-win-02.png
+      :width: 45%
+      :alt: Shrink C
+
+      Figure 2: Shrink C.
+
+#. Shutdown the Windows 10 OS.
+
+#. Follow the instructions to :ref:`bare-metal-install-desktop`.
+
+#. After booting from the |CL| image, select the icon to
+   launch the installer.
+
+#. Click :guilabel:`Select Installation Media`.
+
+#. Select :guilabel:`Safe Installation`.
+
+#. Go through the remaining steps to install |CL|.
+
+#. Shut down your system and remove the USB.
+
+#. Reboot.
+
+#. During the BIOS POST stage, press :kbd:`F10`, or the proper F-key for your
+   system, to launch the :guilabel:`Boot Menu`.
+
+   .. figure:: ../../_figures/multi-boot/multi-boot-win-03.png
+      :width: 45%
+      :alt: Boot menu
+
+      Figure 3: Boot menu
+
+#. In the :guilabel:`Boot Order` menu, use the arrow to select the
+   :guilabel:`Linux bootloader` as boot device.
+
+   Some BIOSes do not support listing multiple partitions. In this case,
+   it will only show one bootable partition.
+
+   .. tip::
+
+      If you don't want to use the BIOS boot menu each time to select an OS,
+      follow :ref:`Advanced: Use systemd-boot to boot Windows 10 OS and |CL| <advanced-systemd-boot>`.
+
+Method 2: Add another hard disk to your system where you install |CL|
+*********************************************************************
+
+#. Shutdown your system.
+
+#. Open your system and attach another hard drive.
+
+#. Power up your system.
+
+#. Follow the instructions in :ref:`bare-metal-install-desktop`, and launch
+   the |CL| installer.
+
+#. In the :guilabel:`Required options` tab, choose :guilabel:`Select
+   Installation Media`.
+
+#. Within that menu, select :guilabel:`Destructive Installation`, and
+   select the new hard drive from the device list.
+
+   .. note::
+
+      Make sure you don’t select the drive with your Windows 10 OS.
+
+#. Go through remaining steps to complete the installation.
+
+#. Reboot.
+
+#. During the BIOS POST stage, press :kbd:`F10`, or the proper F-key for your
+   system, to launch the :guilabel:`Boot Menu`.
+
```

Comment:
```suggestion
   .. figure:: ../../_figures/multi-boot/multi-boot-win-03.png
      :width: 45%
      :alt: Boot menu

      Figure 4: Boot menu
```

Reply:
@mvincerx - what do you think about replicating Figure 3 here for Method 2?

Reply:
Sure. User confirmation with screenshots is always good. Upcoming.